tests/test_uds.py

```diff
@@ -61,37 +82,90 @@ def setup_graph():
     pp_graph = PredPattGraphBuilder.from_predpatt(pp, ud, 'tree1')
 
     graph = UDSGraph(pp_graph, 'tree1')
-    graph.add_annotation(*ann1['tree1'])
-    graph.add_annotation(*ann2['tree1'])
+    graph.add_annotation(*node_ann['tree1'], raw_properties=node_raw_properties)
+    graph.add_annotation(*edge_ann['tree1'], raw_properties=edge_raw_properties)
     
     return graph
 
-# def setup_corpus():
-#     rawfile = StringIO(rawtree)
-#     return UDSCorpus.from_file(infile=rawfile)
+def test_load_uds_corpus():
+    # TODO
+    # Remove corpus JSON files to test start-up load
+    # os.system(f'rm {CORPUS_DIR}*.json')
+
+def test_constructing_rdf_for_graph_with_raw_annotations_fails():
+    graph = setup_graph(normalized=False)
+    assert hasattr(graph, '_rdf') == False # RDF not yet built
+    assert graph.raw_properties == {'property'} # single raw property
+    assert_raises(TypeError, lambda: graph.rdf) # attempt to build RDF
+
+def test_direct_instantiation_of_uds_annotation_fails():
+    assert_raises(TypeError, UDSAnnotation, None)
```

Comment:
Still need to add a few tests to feel comfortable with these changes, but I've written a number of the most important ones and fixed the one that was failing below.